nixos/modules/services/printing/cupsd.nix

```diff
@@ -185,6 +190,17 @@ in
         '';
       };
 
+      printersConf = mkOption {
+        type = types.nullOr types.lines;
+        default = null;
+        description = ''
+          Contents of the printers configuration file of the CUPS daemon
+          (<filename>printers.conf</filename>). When left null, cups will use
+          the (<filename>/etc/cupsd/printers.conf</filename>) as the configuration
```

Comment:
Maybe `/etc/cups/` was meant here? Also CUPS will use this path anyway, it's just that the file will or won't be there.